repos.json

```diff
@@ -140,6 +140,7 @@
     { "language": "nodejs", "repo": "googleapis/nodejs-dialogflow", "apiHint": "dialogflow"},
     { "language": "nodejs", "repo": "googleapis/nodejs-dlp", "apiHint": "dlp"},
     { "language": "nodejs", "repo": "googleapis/nodejs-dns", "apiHint": "dns"},
+    { "language": "nodejs", "repo": "googleapis/nodejs-document-ai"},
```

Comment:
we should add the API hint, it should be the prefix of the API endpoint that the library interacts with, probably something like `documentai`; you can find in the `client.ts` file, or protos.

Reply:
Done

Reply:
I am so sorry.  Turns out I deleted all API hints in a separate PR.  It was a vestigial limb.

Reply:
Removed.

NP, I understand. Thank you both for all your help.